convert to C.HLOCAL via unsafe.Pointer

josharian opened the pull request:
See https://github.com/golang/go/issues/51726 for discussion.

josharian committed:
convert to C.HLOCAL via unsafe.Pointer

See https://github.com/golang/go/issues/51726 for discussion.

josharian commented:
@dblohm7 can I ask you to review this? All the context you need is in the linked issue. I was hoping that you in particular could review, as you're probably the best placed to ensure that we understand Windows half of this interaction. If you'd like to discuss the Go half at all, ping me. Thanks!

josharian committed:
.github/workflows: update Go versions

We really only care about supporting Go 1.18+,
but leaving 1.17 in there doesn't hurt.

dblohm7 reviewed (approved):
LGTM. I agree with the assessment in the referenced Go issue and will add a bit of additional context there.

josharian committed:
remove SHA1 support

It is considered broken.
Go 1.18 no longer supports it.
We don't use it.
Remove it.
